var/spack/repos/builtin/packages/damaris/package.py

```diff
@@ -87,6 +92,13 @@ def cmake_args(self):
 
         if self.spec.variants["python"].value:
             args.extend(["-DENABLE_PYTHON:BOOL=ON"])
+            args.extend(["-DENABLE_PYTHONMOD:BOOL=ON"])
+            args.extend(
+                [
+                    "-DPYTHON_MODULE_INSTALL_PATH:PATH=%s/lib/python%s/site-packages"
+                    % (self.prefix, self.spec["python"].version.up_to(2).dotted)
+                ]
+            )
```

Comment:
```suggestion
            args.append(self.define("PYTHON_MODULE_INSTALL_PATH", python_platlib))
```
See if this works. The directory you used is not always correct on every platform. Sometimes it's lib64, sometimes it's dist-packages, etc.

Reply:
Great, i'll give that a shot. For Spack, this will override the CMake logic.